### DISCUSSION TITLE
When build , 'Component' cannot be used as a JSX component error in  _app.tsx

### DISCUSSION THREAD

**Discussion by EhanDuan** [chakra-ui/chakra-ui#6212] [Q&A] [answered]
Hi, I am using `nextjs` to develop an app.

When I build the project, it show  the following error.
```
Component' cannot be used as a JSX component error 
```

In the page, it is also marked as error. But everything works current well when `npm run dev`.
<img width="839" alt="image" src="https://user-images.githubusercontent.com/75397936/174452303-14637b60-d0c3-4a44-8397-bf0efdc26f88.png">

<img width="1218" alt="image" src="https://user-images.githubusercontent.com/75397936/174452335-5b78c607-ecdf-4bf3-af99-cf4411feb2b1.png">

Thanks for your help!

**Comment by TylerAPfledderer** [comment, marked as answer]
Hey @EhanDuan!

To verify, have you remade the lock file and made sure all your type dependencies are up-to-date? I've experienced this once before and one of the common culprits is conflicting versions of type packages (or maybe missing @types packages)

**Reply by EhanDuan** [reply]
Hi @TylerAPfledderer 

Does it mean I need to run `npm install` to keep them are up-to-date?
> remade the lock file and made sure your type dependencies are up-to-date?

**Reply by TylerAPfledderer** [reply]
You can check using `npm outdated` to see what has a newer version.

What I would do first though is clean by removing `node_modules` and your `package-lock.json` and then `npm i` to do a fresh install to ensure versions are synced. Then from there you can check for said updates as needed and ensure you have all the `@types` packages that are required.

**Reply by EhanDuan** [reply]
Thanks! In addition, would these operations change other codes, like in `src` folders?
> removing `node_modules` and your `package-lock.json` and then `npm i` to do a fresh install to ensure versions are synced.

**Reply by TylerAPfledderer** [reply]
No, package-lock records versions of the 3rd party packages and their dependencies as they are currently installed. This prevents you from possibly installing newer versions of the packages by mistake if you have to clone the repo again and reinstall everything (this is based on what `package.json` has marked for each dependency version). This is also used to help determine what packages need updating. 

The `node_modules` folder simply contains all the files needed for the dependencies you install to be used in your projects. 

There are other uses for each one, but nothing else in this project should really change. 

Worst case is if you run into any problems, you can revert the changes to the lock file and any other unintended changes via git. 😁

**Reply by EhanDuan** [reply]
Great! Thanks for your help! I will try that soon.
